syft/sandbox.py

```diff
@@ -1,22 +1,20 @@
 import importlib
 
-import torch
-
 from syft.frameworks.torch.hook.hook import TorchHook
 from syft.workers.virtual import VirtualWorker
 from syft.grid.private_grid import PrivateGridNetwork
 
 from syft.exceptions import DependencyError
 
 
-def create_sandbox(gbs, verbose=True, download_data=True):
+def create_sandbox(gbs, verbose=True, download_data=True):  # noqa: C901
     """There's some boilerplate stuff that most people who are
     just playing around would like to have. This will create
     that for you"""
 
     try:
         torch = gbs["torch"]
-    except:
+    except:  # noqa: E722
```

Comment:
What exceptions would we expect here? `KeyError`?

Reply:
Yes :) -- will change it with that